shared/js/background/chrome-events.es6.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * NOTE: this needs to be the first listener that's added
+ *
+ * on FF, we might actually miss the onInstalled event
+ * if we do too much before adding it
+ */
+const ATB = require('./atb.es6')
```

Comment:
We discussed using a different event interface, (`EventEmitter2`, same as the UI code) but I thought a bit more about it, and reckoned that still opened up the possibility for us missing the oninstalled event. I think having it here, with comments here and in the entry point, is more explicit.

Plus, the "outside-in" approach for calling the methods on each of the modules might make testing easier.